Update CHANGELOG.txt

StephanStS opened the pull request:
<!--
Before submitting a pull request:
- Please ensure the target branch is "dev" (active development): https://github.com/MichaIng/DietPi/tree/dev
- Please ensure changes have been tested and verified functional.
-->
**Status**: Ready

**Reference**: https://github.com/MichaIng/DietPi-Docs/issues/271

**Commit list/description**:
- File CHANGELOG.txt | Text improvements:
  - Links to DietPi-docs added
  - "New Software Packages" grouped
  - "Removed Software Packages" grouped

@MichaIng: A few further links can be added in the file if the next DietPi-Docs changes are on the DietPi-Docs master branch. I will keep an eye on this.

StephanStS committed:
Update CHANGELOG.txt

- Links to DietPi-docs added
- "New Software Packages" grouped
- "Removed Software Packages" grouped

MichaIng commented:
Ah sorry Stephan, but this is a regular text file only, not a Markdown file. I understood the idea that we add those to the PR's text: https://github.com/MichaIng/DietPi/pull/3960
I'll do it like that:
- Copy&paste your changes into the PR: ...done: https://github.com/MichaIng/DietPi/pull/3960
- Copy&paste the PR text back to CHANGELOG.txt, which should exclude the links 😄... arrg, GitHub internal links are shortened then as well and the list dash is missing 🤔. Let me do the reordering manually, that is not a big deal. ...done.

MichaIng committed:
v6.34

+ CHANGELOG | Remove docs links from plain text file, add Portainer and Tor Relay to "new software titles", move OpenBazarr back to "changes", rename "Single File PHP Gallery" back to "Single File PHP Gallery", as this it it's name and should be renamed to that in the docs as well.

MichaIng committed:
v6.34

+ CHANGELOG | Remove docs links in outstanding issues
